app/src/main/java/com/simplecity/amp_library/ui/screens/nowplaying/PlayerFragment.java

```diff
@@ -334,6 +334,11 @@ public void onResume() {
                             uiEventType = UiEventType.SEEK_START;
                             isSeeking = true;
                         } else if (seekBarChangeEvent instanceof SeekBarStopChangeEvent) {
+                            /*
```

Comment:
In-line code comments that span multiple lines should start with `//` on each line instead of being in a block comment.